Formula/usbutils.rb

```diff
@@ -0,0 +1,37 @@
+class Usbutils < Formula
+  desc "Provides 'lsusb' tool for getting detailed info about USB devices"
+  # Homepage for multiple Linux USB tools, 'usbutils' is one of them.
+  homepage "https://linux-usb.sourceforge.io"
+  # The latest version we can use is 007 (from 2013), because the later
+  # versions depend on 'libudev': part of Systemd and not in Homebrew.
+  url "https://mirrors.edge.kernel.org/pub/linux/utils/usb/usbutils/usbutils-007.tar.xz"
+  sha256 "7593a01724bbc0fd9fe48e62bc721ceb61c76654f1d7b231b3c65f6dfbbaefa4"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "pkg-config" => :build
+  depends_on "libusb"
+
+  def install
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+    # Clear/override several variables, to avoid building/installing
+    # linux-specific utilities that rely on sysfs.
+    system "make", "install", "SUBDIRS=", "AM_LDFLAGS=", "bin_SCRIPTS=",
```

Comment:
Why is `AM_LDFLAGS` there? I don't think it has to do with sysfs

Reply:
Inaccurate comment. Updated. Please take another look.
AM_LDFLAGS is needed because:

usbutils-007/Makefile.am, line 4:

AM_LDFLAGS = \
	-Wl,--as-needed

The MacOS linker does not support "--as-needed" flag, so we need to override it from the command line to avoid the linker errors. Macports does the same, BTW.